src/lib/domain/screener/engine.ts

```
@@ -0,0 +1,402 @@
+	if (growthPct == null) return { engine: 'N/A', signal: 'NO_DATA', note: 'Missing growth data' };
+
+	const isPreProfit = model?.ttmEPS != null && model.ttmEPS <= 0;
+	if (isPreProfit) {
+		return { engine: 'N/A', signal: 'NO_DATA', note: 'Pre-profit; screener not applicable' };
+	}
+
+	if (growthPct > GROWTH_ROUTING_THRESHOLD) {
+		const divYieldPct = parsePercent(model?.dividendYield) ?? 0;
+		const isAltAsset = stock.group === 'Financials & Alt Assets' || /fre|ani|fee-related|distributable/i.test(model?.basis ?? '');
+		
+		if (divYieldPct < 5.0 || isAltAsset) {
```

ESLint

[error] Don't use a zero fraction in the number.

(unicorn/no-zero-fractions)